phys/module_sf_urban.F

```diff
@@ -1583,15 +1671,25 @@ SUBROUTINE urban(LSOLAR,                                           & ! L
    PSIX10 = ALOG(10./Z0) - PSIM10
    PSIT10 = ALOG(10./Z0H) - PSIH10
 
+   IF (distributed_aerodynamics_option .and. Z0C > 10.) THEN
+     U10 = 0.
```

Comment:
@epn09 not sure if this is a good assumption. maybe still use the logscale decay? Is this assumption related to how you treat the roughness from your new input parameters?

Reply:
@cenlinhe Currently in this parameter database that we have, there is no grid with z0 higher than 10 m, so this if statement actually will never be executed. But the treatment of grid with z0 > 10 m (if exists) is something we also don't know how to properly handle because in my understanding log-law is not valid for z < z0. So if z = 10 m and z0 > 10 m, I don't really know what should be done. In the mean time, this if statement can be removed. Please let me know your opinion.

Reply:
I do not have a good solution to this either. The default treatment is using logscale decay. For Z0C > 10, the log law should give a relatively small value (close to 0) at z=10. Maybe for now, just follow the default treatment.

Reply:
@cenlinhe This is fixed. The if Z0C > 10 is removed.